kombu/transport/consul.py

```diff
@@ -276,24 +276,25 @@ class Transport(virtual.Transport):
     driver_type = 'consul'
     driver_name = 'consul'
 
-    def __init__(self, *args, **kwargs):
-        if consul is None:
-            raise ImportError('Missing python-consul library')
-
-        super().__init__(*args, **kwargs)
-
-        self.connection_errors = (
+    if consul:
```

Comment:
only 3 transports are subject to this adjustments?

Reply:
Yeah, I did a search for `self.connection_errors` and `self.channel_errors`, and looked at each of the transports.

For some reason, the librabbitmq and mongodb backends don't have the `try/except ImportError:` logic and they don't use `__init__` to fill in `connection_errors` and `channel_errors`.